modules/net-glb/backend-services.tf

```diff
@@ -21,10 +21,14 @@ locals {
     for k, v in coalesce(var.backend_services_config, {}) :
     k => v if v.bucket_config != null
   }
-  backend_services_group = {
+  backend_services_group = var.region == null ? {
     for k, v in coalesce(var.backend_services_config, {}) :
     k => v if v.group_config != null
-  }
+  } : {}
+  regional_backend_services_group = var.region != null ? {
```

Comment:
nit: can we use a single local, and apply the condition where it's set in the resource?

Reply:
Fixed!